website/docs/guides/visual-editor-qs.md

```diff
@@ -0,0 +1,252 @@
+
+:::tip
+
+Your work in the Visual Editor is automatically saved as you progress, so if you need a break, you can always come back to a session later. Just be sure to give it a unique title!
+
+:::
+
+## Enhance your model
+
+You've got the basics going with your Visual Editor model! It has successfully joined two pre-existing models, but you need to transform the data further to get what you need. A list of customers who buy repeat items as you consider a loyalty club rewards program.
+
+### Aggregate data
+
+Multiple options for transforming your data include custom formulas, filters, and unions. Keep it simple and add an aggregation operator to tell you which customers buy the most repeat products.
+
+1. Drag the **Aggregation** operator over to the right of the join.
+2. Connect the aggregation operator to the join operator. 
+3. Click **Configure aggregation** in the **Aggregation tile**.
+4. Click in the **Group by** field and first select `CUSTOMER_ID` then `PRODUCT_ID`.
+5. Configure the next three fields with the following:
+    - **Function:** Count
+    - **Column:** PRODUCT_ID
+    - **Alias:** count_PRODUCT_ID
+
+    <Lightbox src="/img/docs/dbt-cloud/visual-editor/aggregation.png" width="90%" title="The configured aggregation operator tile." />
+
+6. Click the **Play icon** to preview the data. You're starting to see the results you're looking for, but the data is scattered. Let's clean it up a bit more.
+
+:::tip
+
+As your model grows, you can zoom in and out to view your needs. Click and hold in empty canvas space to drag your setup across the screen. Click the **Fit view** icon to see your entire model on the screen. Click the **Auto layout** icon to auto-arrange the tiles efficiently. 
+
+:::
+
+### Add some order
+
+There's a lot of data there. Dozens of customers are buying hundreds of products. You will sort it so that the customers are listed ascending by their CUSTOMER_ID number, with the most purchased products listed in descending order. 
+
+1. Drag the **Order** operator over to the right of the **Aggregation** tile and connec them.
+2. Click **Configure order**.
+3. In the **Sort order** field click **Select column** and click `Aggregation1.CUSTOMER_ID` from the dropdown. Set it to `Asc`. 
+4. Click **Add sorting** and in the new **Select column** field select `Aggregation1.count_PRODUCT_ID`. Set it to `Desc`.
+5. Press the **Play icon** to preview the new data.
+
+    <Lightbox src="/img/docs/dbt-cloud/visual-editor/order.png" width="90%" title="The ordered data operator tile config and data preview." />
+
+:::tip
+
+Want to practice on your own? Try adding a **Filter** operator that removes items with less than 10 sales for any customer ID. Be sure to run the preview and verify the data is correct.
+
+:::
+
+## Run and share your model
+
+Now that you've built a model that results in the data you want, it's time to run it and push it to your Git repo.
```

Comment:
I would break this down and recap how preview/run/prod work at the top, since it's a new concept.
1. preview does not affect state in your warehouse, so you can test and develop.
2. when you're ready to use this mode in a downstream tool, you can run it to materialize it in the warehouse.
3. Once the model is ready to make its way to production and be used by others or orchestrated, you can commit and open a PR.